### ISSUE TITLE
using shinycssloaders in rmd document

### ISSUE THREAD

**Issue by Kvit** [andrewsali/shinycssloaders#11]
is it possible to use shinycssloaders with shiny rmd doc? If so, what are the rules?
I'm trying this example and don't see spinner:

`
---
title: "shinycssloaders"
runtime: shiny
---

```{r setup, include=FALSE}
library(shiny)
library(shinycssloaders)
test_table<-data.frame(A=rnorm(200), B=rnorm(200))

```

```{r}
# should load table with spinner
withSpinner(DT::dataTableOutput("table"))

```

```{r}
output$table<- DT::renderDataTable({
  Sys.sleep(2)
  return(test_table)
})
```
`

**Comment by andrewsali** [comment]
This appears to be a duplicate of #8 . It's on the roadmap to be addressed, the issue is that Rmarkdown with runtime `shiny` doesn't seem to process special tags such as `tags$head` and `singleton`.

**Comment by Kvit** [comment]
In Rmarkdown at least some of this functionality is available via YAML header. 
For example:
---
title: "mytitle"
output:
  html_document:
    includes:
       in_header: my_header.html
---
 will include my_header.html into resulting webpage

Also (in case this is helpful) , you can include shiny functions like: https://shiny.rstudio.com/reference/shiny/latest/addResourcePath.html
in r code chunks with context=setup or context=global  to execute only once

**Comment by daattali** [comment]
@Kvit using shinycssloaders in rmarkdown is now possible
